src/nxdoc/nuxeo-server/administration/kafka.md

```diff
@@ -133,3 +134,4 @@ it is used to send instant message and its retention can be reduced at the Kafka
 ```bash
 $KAFKA_HOME/bin/kafka-configs.sh --zookeeper <zk_host> --alter --entity-type topics --entity-name nuxeo-pubsub --add-config retention.ms=7200000
 ```
+-->
```

Comment:
Ok to keep this section only for 9.10 but we should keep a note related to the pubsub topic configuration where it makes sense to have a lower retention.
for instance we can add:
Note that when Kafka is used as PubSub Provider implementation we don't need to have a 7 days retentions, PubSub is mainly used to send instant message and its retention can be reduced at the Kafka level to 2 hours:
<keep the bash section with kafak-configs.sh command>